Eskillate/Assets/Scripts/LowPop/GameController.cs

```diff
@@ -31,19 +33,72 @@ void Start()
                 LevelId = 2,
                 Name = "Level2",
                 Description = "This is level 2.",
-                HighScore = 0
+                HighScore = highScores[2]
             };
 
             _levels.Add(level1);
             _levels.Add(level2);
 
-            // TODO - Remove this as it will be called from the level selection menu
-            LoadLevel(0);
+            LoadHelper.LoadSceneAdditively(this, "LevelSelectionMenu", FillLevelSelectionMenu);
 
             // Add menus
             LoadHelper.LoadGenericMenus(this);
         }
 
+        private void FillLevelSelectionMenu()
+        {
+            var levelSelectionCanvas = GameObject.Find("LevelSelectionCanvas");
+            var scrollListContent = levelSelectionCanvas.transform.Find("ScrollList").transform.Find("ScrollListViewport").transform.Find("ScrollListContent");
+
+            foreach (Transform child in scrollListContent.transform)
+            {
+                GameObject.Destroy(child.gameObject);
+            }
+
+            int levelCountForThisLevelTrio = 0;
+            GameObject currentLevelTrio = new GameObject();
+            foreach (var level in _levels)
+            {
+                levelCountForThisLevelTrio = levelCountForThisLevelTrio % 3;
+                if (levelCountForThisLevelTrio == 0)
+                {
+                    currentLevelTrio = Instantiate(Resources.Load("Core\\Prefabs\\LevelSelection\\LevelTrio")) as GameObject;
+                    currentLevelTrio.transform.SetParent(scrollListContent.transform, false);
+                }
+                var levelGO = currentLevelTrio.transform.GetChild(levelCountForThisLevelTrio);
+                var scoreGO = levelGO.GetChild(0);
+                scoreGO.GetComponent<TMPro.TextMeshProUGUI>().text = level.HighScore.ToString();
+                var nameGO = levelGO.GetChild(1);
+                nameGO.GetComponent<TMPro.TextMeshProUGUI>().text = level.Name;
+                var starsGO = levelGO.GetChild(2);
+                var selectButtonGO = levelGO.GetChild(3);
+                selectButtonGO.GetComponent<Button>().onClick.AddListener(delegate { OnLevelSelected(level.LevelId); });
+
+                levelCountForThisLevelTrio++;
+            }
+
+            while (levelCountForThisLevelTrio != 3)
+            {
+                GameObject.Destroy(currentLevelTrio.transform.GetChild(levelCountForThisLevelTrio).gameObject);
+                levelCountForThisLevelTrio++;
+            }
+
+            // Set Y pos to 1 to be at scrolled to the top
+            scrollListContent.transform.parent.transform.parent.GetComponent<ScrollRect>().normalizedPosition = new Vector2(0, 1);
+        }
+
+        private void OnLevelSelected(int levelId)
+        {
+            UnloadLevel();
+            LoadLevel(levelId - 1);
```

Comment:
I dont like this. I think the levels ID should reflect their position in the list.